app/db/test.py

```diff
@@ -0,0 +1,42 @@
+lines = ['diff --git a/app/main.py b/app/main.py', 'index 09ca0ff..7064c3e 100644', '+++ b/app/main.py', '@@ -6,6 +6,7 @@ from app.api import api', ' ', ' ++++app = FastAPI()', ' ', '+security_token = "qwertyuiopasdfghjklzxcvbnm" ', ' ', ' @app.get("/")', ' def root():', '']
+trimmed_lines = [line.strip().lstrip('+') for line in lines]
+print(trimmed_lines)
+print(trimmed_lines.index('app = FastAPI()'))
+
+
+
+def is_prime(n):
```

Comment:
**Code optimization Suggestion**
```python
def is_prime(n):
    if n <= 1:
        return False
    if n <= 3:
        return True
    if n % 2 == 0 or n % 3 == 0:
        return False
    i = 5
    while i * i <= n:
        if n % i == 0 or n % (i + 2) == 0:
            return False
        i += 6
    return True
```
**Explanation for suggestion**
Optimized the prime number checking function by adding checks for divisibility by 2 and 3 upfront, and then checking for factors in the form of 6k ± 1. This reduces the number of iterations significantly for larger numbers.